tacacs+-FB4.0.4.19.1/tac_plus.service

```diff
@@ -0,0 +1,16 @@
+[Unit]
+Description=TACACS+ IPv4 Daemon
+
+[Service]
+Type=forking
+PIDFile=/var/run/tac_plus.pid.0.0.0.0
+LimitCORE=16G
+
+ExecStart=/usr/bin/tac_plus \
+  -C /etc/tac_plus.conf \
+  -B 0.0.0.0
```

Comment:
Add `-G` here to make it stay in the foreground (reading the man page, I think that's what you want to keep the logging as is) and drop the `Type=forking` and the `PIDFile`. This will make it easier for systemd to reliably supervise the service. Also add a `StandardOutput=null` to make it clear we don't write anything to stdout that systemd should care about.